backend/box/box.go

```diff
@@ -84,6 +107,13 @@ func init() {
 		}, {
 			Name: config.ConfigClientSecret,
 			Help: "Box App Client Secret\nLeave blank normally.",
+		}, {
+			Name: "config_json",
```

Comment:
is `config_json` what users would call it?  Might `jwt_file` or something like that be better?

Also would users ever put it directly in the config file?

Apologies if these are silly questions, I dont know much about JWT!

Reply:
In the case of Box I think possibly, the "Authenticate with JWT" [tutorial](https://developer.box.com/docs/construct-jwt-claim-manually#section-1-read-json-config) refers to it as `config.json`, and when downloading the file from the box developer page when logged in it saves it as `123456__config.json` (random numbers).

`config.json` does seem rather generic for rclone purposes though. I think `jwt_file` might be misleading as the file doesn't contain the jwt but rather the config to create a jwt assertion which can then be exchanged for an oauth token. Maybe `box_config_file`?